projects/io-backend-projects/provider.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
```

Comment:
```suggestion
terraform {
  required_version = ">= 0.14.5"
```

Reply:
I will also try the newest terraform 0.14.*

Reply:
by the way I think it's better to fix the version .... or use tfenv....

Reply:
thanks, upgraded to 0.14.5. Updated .terraform-version for tfenv